examples/hybrid/sphere/README.md

```diff
@@ -13,7 +13,7 @@ Here is a sbatch script template for setting up simulations on caltech central h
 #SBATCH --output=$YOUR_SIMULATION_LOG_DIR/simulation.log
 
 module purge
-module load julia/1.9.3 openmpi/4.1.1
+module load julia/1.10.0 openmpi/4.1.1
```

Comment:
I think that this snippet might no longer working since we use MPITrampoline now (required for compatibility with new versions of HDF5).

Maybe it is a good time to update it or remove it if we don't want to maintain it.

Reply:
Are you suggesting to change this to `module load julia/1.10.0`?

Reply:
The snipped diverged from what we do in the buildkite pipeline with respect to MPI. I am worried it wouldn't work.

Reply:
It is not in the scope of this PR, but it might be a good opportunity to update it if you want to.

Reply:
Ok, I don't know off hand, I'll tackle that separately

Reply:
Sounds good, feel free to merge